### ISSUE TITLE
Support Drag and Drop a ZIP file to be uploaded to Catapult

### ISSUE THREAD

**Issue by gastaldi** [redhat-kontinuity/catapult#164]
It would be super if we could drag and drop a ZIP file from our local machine to a web page that would upload it to catapult

**Comment by ALRubinger** [comment]
To be implemented in Catapult itself?  Sounds like a UI component that might be embedded anywhere, calling Catapult as a service?

**Comment by gastaldi** [comment]
Yeah, I was thinking of a component similar to http://imgur.com/upload.
